cg/meta/workflow/mutant.py

```diff
@@ -84,19 +81,21 @@ def link_fastq_files(self, case_id: str, dry_run: bool = False) -> None:
                     case=case_obj, sample=sample_obj, concatenate=True
                 )
                 continue
-            if not sample_obj.sequencing_qc:
-                LOG.info("Sample %s read count below threshold, skipping!", sample_obj.internal_id)
+            sample_passed_sequencing_qc: bool = QualityControllerService.sample_pass_sequencing_qc(
+                sample_obj
+            )
+            if not sample_passed_sequencing_qc:
+                LOG.info(f"Sample {sample_obj.internal_id} read count below threshold, skipping!")
```

Comment:
also rename `sample_obj` to `sample` and `case_obj` to `case`
```suggestion
            if not QualityControllerService.sample_pass_sequencing_qc(sample):
                LOG.info(f"Sample {sample_obj.internal_id} read count below threshold, skipping!")
```

Reply:
I wont do that here, I think I have already started to touch too many files, but perhaps another PR can be opened to fix any mentions of the `case_obj` and `sample_obj`